include/modem/lte_lc.h

```diff
@@ -214,9 +214,10 @@ int lte_lc_normal(void);
  * to be used when psm mode is subsequently enabled using `lte_lc_psm_req`.
  * For reference see 3GPP 27.007 Ch. 7.38.
  *
- * @param rptau Requested periodic TAU.
- * @param rat Requested active time
- *
+ * @param rptau Requested periodic TAU as null-terminated string.
+ *        Set NULL to use manufacturer-specific default.
```

Comment:
I'm actually not sure if the default is specific to the manufacturer (Nordic) or the network operator. @tokangas Do you happen to know?

Reply:
It's the parameters to be used in ATTACH_REQUEST/TAU_REQUEST and network decide the actual parameters in ATTACH_ACCEPT/TAU_ACCEPT. What we want is to use modem-default parameters in ATTACH_REQUEST/TAU_REQUEST

Reply:
Oh right, no I understand the context. Then it makes good sense to keep as is, thanks for explaining.